src/main/java/com/nu/art/cyborg/modules/notifications/NotificationHandler.java

```diff
@@ -56,7 +58,12 @@ protected final void cancelNotification(short notificationId) {
 	}
 
 	protected final Notification postNotification(Builder builder, short notificationId) {
-		return module.postNotification(builder, notificationId);
+		Notification notification = module.postNotification(builder, notificationId);
```

Comment:
Please separate into two distinct errors:

One the notification doesn't have a channel.
The notification manager doesn't declare a channel.

Reply:
Please use the exception generator object to create the error.

Reply:
Added 2 separate ad hoc methods in ExceptionGenerator with explanation.